demos/python_demos/multi_camera_multi_person_tracking/utils/network_wrappers.py

```diff
@@ -11,10 +11,29 @@
  limitations under the License.
 """
 
+import json
+import logging as log
+from collections import namedtuple
+from abc import ABC, abstractmethod
+
+import cv2
+import numpy as np
+
 from utils.ie_tools import load_ie_model
+from .segm_postprocess import postprocess
+
+
+class DetectorInterface(ABC):
+    @abstractmethod
+    def run_asynch(self, frames, index):
```

Comment:
`async` instead of `asynch` is better:
1. It is a common way to abbreviate the word like that
2. This is how dldt abbreviates it

Reply:
Fixed.